composer.json

```diff
@@ -87,13 +87,16 @@
     },
     "require-dev": {
         "jackalope/jackalope-doctrine-dbal": "^1.2.5",
+        "league/flysystem": "^1.0",
         "matthiasnoback/symfony-config-test": "^3.0 || ^4.0",
         "matthiasnoback/symfony-dependency-injection-test": "^2.0 || ^3.0",
         "php-ffmpeg/php-ffmpeg": "^0.13",
         "phpcr/phpcr-shell": "^1.0",
         "phpdocumentor/reflection-docblock": "~3.1",
         "phpspec/prophecy": "^1.7",
         "phpunit/phpunit": "^6.5",
+        "superbalist/flysystem-google-storage": "^7.1",
+        "google/cloud-storage": "~1.0",
```

Comment:
maybe we should think about running a test suite where optional dependencies are not installed and specific tests skipped to make sure that they are really optional /cc @sulu/core-team